File: datasets/JPlag-Java/to-kotlin-convertai/Verbose.kt
package java

import java.util.ArrayList

object Verbose {
    fun tryCount(l: ArrayList<Int>, n: Int): Int {
        if (l.isEmpty()) {
            return Int.MAX_VALUE
        }

        var count = 0

        for (i in 0 until l.size - 1) {
            if (l[i] + 1 != l[i + 1]) {
                count++
            }
        }
        if (l.first() != 0) {
            count++
        }

        if (l.last() != n - 1) {
            count++
        }

        return count
    }
}

File: datasets/JPlag-Java/to-python-convertai/Verbose.py
import sys

def try_count(l, n):
    if not l:
        return sys.maxsize
    
    count = 0
    
    for i in range(len(l) - 1):
        if l[i] + 1 != l[i + 1]:
            count += 1
    
    if l[0] != 0:
        count += 1
    
    if l[-1] != n - 1:
        count += 1
    
    return count

